cloudvolume/frontends/graphene.py

```diff
@@ -261,6 +262,33 @@ def get_roots(self, segids, timestamp=None, binary=True, stop_layer=None):
 
     return np.array(roots, dtype=self.meta.dtype)
 
+  def get_seg_ids_in_chunk(self, chunk_id, timestamp=None):
+    """
+    Get all of the segments in a lvl 2 chunk,
+    as well as their associated supervoxels.
+
+    chunk_id: np.uint64 lvl 2 chunk id
```

Comment:
I think we need to be careful to describe what a chunk_id is. In the rest of this section, a chunk_id is defined as given a label <LAYER,X,Y,Z,SEGID>, the chunk_id is uint64(X << 20 |Y << 10 | Z) (assuming 10 is the offset for this graph). I noticed in some other cost samples you gave me, you defined the chunkid as meta.encode_label(LAYER,X,Y,Z,segid=0). I'm assuming here we are using the latter? Should we reconcile these definitions?

Reply:
Talked about this with Manuel. Chunk ID is defined as LAYER/X/Y/Z/0 (ie segid zeroed out). I'll call the chunk_id I was using `chunk_position_number`